utils/responsewriter/README.md

```diff
@@ -0,0 +1,29 @@
+# responsewriter
+
+This package contains code from [`github.com/urfave/negroni`](https://github.com/urfave/negroni).
```

Comment:
Why can not import directly? Do we change some codes? If so, please comment changes.

Reply:
Yes there are significant changes, such as support for user values (which we need when moving from fasthttp - may be removed in the future) and removal of "features" (that negroni only needs): https://github.com/dapr/dapr/pull/6248/files#diff-83f49a9138e9ad18fbee03f55f1035fa5ca1a626e1777ec037752212db0e11b7R74

Also, I would prefer not import the entire negroni framework if we just need a single struct. Importing the entire package would cause a lot more code to be brought in that we don't need.

Reply:
Got, please comment changes in the README.

Reply:
Done